features/fixtures/ios/build.sh

```diff
@@ -5,8 +5,44 @@ echo "$(sw_vers -productName) $(sw_vers -productVersion) $(sw_vers -buildVersion
 
 set -euxo pipefail
 
+disable_swizzling_key='<key>test_key_to_replace1</key>'
+disable_swizzling_value='<string>test_value_to_replace1</string>'
+swizzling_premain_key='<key>test_key_to_replace2</key>'
+swizzling_premain_value='<string>test_value_to_replace2</string>'
+
+fixture_name='Fixture'
+for ((i=1;i<=$#;i++));
+do
+    if [ ${!i} = '--fixtureName' ]
+    then ((i++))
+        fixture_name=${!i};
+    fi
+    if [ ${!i} = '--disableSwizzling' ]
+    then
+        disable_swizzling_key='<key>disableSwizzling</key>'
```

Comment:
So there are Plist keys `disableSwizzling` and `swizzleViewLoadPreMain` that can independently be set to `true`?  What happens if someone puts both keys in set to true?

Would it be better to have a single Plist key called something like `swizzlingMode` that can be set to either `default`, `premain` or `disabled`?

Reply:
That's a product decision, so it's outside of the scope of this PR. I will bring this up with the team. For now it has to stay as is.